docs/log-correlation.asciidoc

```diff
@@ -35,14 +35,20 @@ See the relevant https://www.elastic.co/guide/en/apm/agent/index.html[Agent docu
 
 Once log correlation is enabled,
 you must ensure your logs contain APM identifiers.
+
+Log correlation relies on these fields:
+- Service level: {ecs-ref}/ecs-service.html[`service.name`], {ecs-ref}/ecs-service.html[`service.version`], and {ecs-ref}/ecs-service.html[`service.environment`]
+- Trace level: {ecs-ref}/ecs-tracing.html[`trace.id`] and {ecs-ref}/ecs-tracing.html[`transaction.id`]
+
 In some supported frameworks, this is already done for you.
 In other scenarios, like for unstructured logs,
 you'll need to add APM identifiers to your logs in any easy to parse manner.
 
-The identifiers we're interested in are: {ecs-ref}/ecs-tracing.html[`trace.id`] and
-{ecs-ref}/ecs-tracing.html[`transaction.id`]. Certain Agents also support the `span.id` field.
+Log correlation relies on these fields:
+- Service level: {ecs-ref}/ecs-service.html[`service.name`], {ecs-ref}/ecs-service.html[`service.version`], and {ecs-ref}/ecs-service.html[`service.environment`]
+- Trace level: {ecs-ref}/ecs-tracing.html[`trace.id`] and {ecs-ref}/ecs-tracing.html[`transaction.id`]
```

Comment:
I might be missing something here, but isn't this section an exact duplication of https://github.com/elastic/apm-server/pull/10454/files#diff-5e2162539a2b2368823380468a7d1c228a464c989ba793fe81e7e36fabffbf1cR39-R42 - why are we mentioning this twice here?

Reply:
good catch, duplication removed in f4b8913994e38e749e652247adf7866f53614d77